### PR TITLE
bug: fixed how click events are fired #3493

### PR DESCRIPTION
**Pull request by Tzyinc**
**Description:**
from #3493 added distance calculation for the two points mousdown and mouse up

**Changes proposed:**
-
-
-

### CODE REVIEW

File: src/components/cursor.js [hunk 1]
```diff
@@ -26,6 +26,9 @@ var CANVAS_EVENTS = {
 
 var CANVAS_HOVER_CLASS = 'a-mouse-cursor-hover';
 
+// check cursor move over this treshold
+var CURSOR_MOVE_TRESHOLD = 50;
```
Comment (vincentfretin) [inline src/components/cursor.js]: missing a h => threshold

Comment (Tzyinc) [reply src/components/cursor.js]: sorry for the late reply! will make a change over the weekend! also will fix the  CI errors

Comment (Tzyinc) [reply src/components/cursor.js]: @vincentfretin after trying to do this over this weekend, I think I still need some guidance.

I added a  check in onCursorUp, to not trigger EVENTS.CLICK  if the distance is greater than the treshold, but i'm still failing the CI as evt can be undefined. is there a better way for me to get x and y of the cursor at any given time?

**Comment by dmarcos** [comment]
Closing due to inactivity. Can reopen if necessary!

**Pull request closed without merging**